pylint_odoo/test_repo/broken_module2/tests/data/odoo_data_noupdate_1.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+    </data>
+    <data>
+    </data>
+</odoo>
```

Comment:
@dreispt 
Sorry, I don't understand
The file [broken_module2/tests/data/help_test_data.xml](https://github.com/OCA/pylint-odoo/blob/master/pylint_odoo/test_repo/broken_module2/tests/data/help_test_data.xml) has a `<data>` tag (only one) inside another tag `<odoo>`. That is the purpose of this check

The files with the examples were saved in `broken_module2` and `broken_module` only to use the same existing folders.

There is no special reason